### PR TITLE
Update kgtk linking

### LINKED ISSUE #47: Update WikiData to use newer version of Linking
Currently, we are using an outdated version of the wikidata linking. This should be updated with the latest version of the code from Mitchell.

### PR DESCRIPTION
**Pull request by elizlee**
Closes #47 
Uses the latest updates from Mitchell's KGTK linking code

### CODE REVIEW

**Comment by joecummings** [comment]
Do we have any stats on how much this improves the linking?

File: cdse_covid/semantic_extraction/run_wikidata_linking.py [hunk 1]
```diff
@@ -86,22 +109,35 @@ def get_best_qnode_for_mention_text(
     # If no Qnode was found, try KGTK
     query_list: List[Optional[str]] = [mention.text, variable_node_label, *claim_variable_tokens]
     for query in list(filter(None, query_list)):
-        claim_variable_links = find_links(claim.claim_sentence, query)
+        claim_variable_links = find_links(
```
Comment (joecummings) [inline cdse_covid/semantic_extraction/run_wikidata_linking.py:112]: It doesn't look like we are ever using "VERB"; however, I would imagine we want to do events.

File: cdse_covid/semantic_extraction/run_entity_merging.py [hunk 2]
```diff
@@ -84,7 +89,7 @@ def find_knowledge_entity(
 
 def create_wikidata_qnode_from_id(mention: Mention, qnode_id: str) -> Optional[WikidataQnode]:
     """Get the rest of the qnode data from KGTK."""
-    cache_file = make_cache_path(KGTK_CACHE, qnode_id)
+    cache_file = make_cache_path(KGTK_REFVAR_CACHE, qnode_id)
```
Comment (joecummings) [inline cdse_covid/semantic_extraction/run_entity_merging.py:92]: Again are we only doing REFVAR's or VERBs'
?

Comment (elizlee) [reply cdse_covid/semantic_extraction/run_entity_merging.py:92]: > Do we have any stats on how much this improves the linking?

Compared with the results from https://github.com/joecummings/cdse-covid/pull/99 where I compared 20 claims:
* The changes correctly identify 7 more x-variable qnodes; 4 of these are due to the improvements from using the entity mapping (#104).
* Each version correctly identified one claimer, but this claimer was different in each.
* No change in event qnodes
* No change in args + role
* Ignoring argument role accuracy, the changes here found two more correct argument qnodes.

Comment (elizlee) [reply cdse_covid/semantic_extraction/run_entity_merging.py:92]: I only used the refvar cache within `create_wikidata_qnode_from_id` because this method creates qnodes for x-variables, claimers, and arguments. I could also have it try the verb cache since it's possible that some of these mentions could have qnodes from there.

Comment (joecummings) [reply cdse_covid/semantic_extraction/run_entity_merging.py:92]: Maybe I'm confused, but I think we want the updated verb linking for the event we find in the semantic information.

File: cdse_covid/semantic_extraction/run_wikidata_linking.py (hunk 1 above)
Comment (elizlee) [reply cdse_covid/semantic_extraction/run_wikidata_linking.py:112]: The assumption made here was that if graph label wasn't a propbank label, then the thing we're querying isn't an event, but I see now how an "event" would reach this point, so I'll add a query for that

File: cdse_covid/semantic_extraction/run_entity_merging.py (hunk 2 above)
Comment (elizlee) [reply cdse_covid/semantic_extraction/run_entity_merging.py:92]: We do run the verb linking here: https://github.com/joecummings/cdse-covid/blob/c3ca3fccb1e56c629cb9737865091b2a296d0692/wikidata_linker/get_claim_semantics.py#L340

**Review by joecummings — COMMENTED** [review]
This looks good, but the `state_dict` should probably be uploaded to Google Drive and mentioned in the README.
